templates/Gemfile.erb

```diff
@@ -41,6 +40,7 @@ group :development, :test do
   gem "pry-byebug"
   gem "pry-rails"
   gem "rspec-rails", "~> 3.4.0"
+  gem "refills"
```

Comment:
Suspenders' test suite itself needs `refills` in the `test` block.

Debugging trick for the test suite:

```diff
diff --git a/bin/suspenders b/bin/suspenders
index 787d6af..e0478a4 100755
--- a/bin/suspenders
+++ b/bin/suspenders
@@ -20,4 +20,14 @@ templates_root = File.expand_path(File.join("..", "templates"), File.dirname(__F
 Suspenders::AppGenerator.source_root templates_root
 Suspenders::AppGenerator.source_paths << Rails::Generators::AppGenerator.source_root << templates_root
 
+class Thor
+  module Shell
+    class Basic
+      def stdout
+        $stderr
+      end
+    end
+  end
+end
+
 Suspenders::AppGenerator.start
```

(I'd like to get that, or something more clear than that, into `master`, hidden behind e.g. `ENV['DEBUG_SUSPENDERS_TO_STDERR']`.)